docs/jwst/user_documentation/running_pipeline_python.rst

```diff
@@ -529,3 +529,196 @@ individual step parameter must be set when using this method, or else the coded
 defaults will be used, which may be inappropriate for the dataset being processed.
 
 See :ref:`call_examples` for more information.
+
+
+.. _multiprocessing:
+
+Multiprocessing
+===============
+
+Python's multiprocessing module explicitly imports and executes a script's
+`__main__` with each and every worker. If `__main__` is not present the behavior is
+undefined. Hence, Python will crash unless the multiprocess code in enclosed in a
+`__main__` block like this:
+
+
+::
+
+    import sys
+
+    def main():
+        [code used in multiprocessing]
+
+    if __name__ = '__main__':
+        sys.exit(main())
+
+
+There are three scenarios to use multiprocessing with the pipeline:
+
+1. Multiprocessing with a pipeline step. At the moment, these steps are
+:ref:`jump_step <jump_step>`, :ref:`ramp_fitting_step <ramp_fitting_step>`,
+and :ref:`wfss_contam_step <wfss_contam_step>`. To enable multiprocessing the
+optional parameters are `max_cores` for the ``jump`` step, and `maximum_cores`
```

Comment:
this is an aside, but I wonder if we should consider changing these to all be the same

Reply:
I agree. Given that jump appears to be the outlier (so to speak), I suggest changing the name `max_cores` to `maximum_cores` in the jump step definitions. Whatever internal names may be used can be left alone.